File: Shared/Blog.Contracts/V1/Requests/TagsRequests/TagRequest.cs
namespace Blog.Contracts.V1.Requests.TagsRequests
{
    using Blog.Contracts.V1.Requests.Interfaces;

    /// <summary>
    /// Tag request.
    /// </summary>
    public class TagRequest : IRequest
    {
        /// <summary>
        /// Gets or sets the title.
        /// </summary>
        /// <value>
        /// The title.
        /// </value>
        public string Title { get; set; }
    }
}

File: Shared/Blog.FSharp.Contracts/V1/Requests/TagsRequests/TagRequest.fs
namespace Blog.FSharp.Contracts.V1.Requests.TagsRequests

open Blog.FSharp.Contracts.V1.Requests.Interfaces

/// <summary>
/// Tag request.
/// </summary>
type TagRequest() =
    interface IRequest

    /// <summary>
    /// Gets or sets the title.
    /// </summary>
    member val Title : string = null with get, set